vue-component/dev/serve.vue

```diff
@@ -1,17 +1,84 @@
 <script lang="ts">
 import { defineComponent } from 'vue';
 import CrossfadeImageViewer from '@/crossfade-image-viewer/crossfade-image-viewer.vue';
+import CrossFadeGallery from '@/cross-fade-gallery/CrossFadeGallery.vue';
+import { ImageCollection } from '../../core/cross-fade-viewer-logic';
 
 export default defineComponent({
   name: 'ServeDev',
   components: {
-    CrossfadeImageViewer
-  }
+    CrossfadeImageViewer,
+    CrossFadeGallery
+  },
+  setup() {
+    let availableImages: ImageCollection = [
+      {title: 'Some Title A', sourceUrl: 'https://i.picsum.photos/id/494/1500/1800.jpg?hmac=rJgjnXxtUmsZnV0nNVlj5F1722L_q7xPXd8VZCuFIfg', opacity: 1},
```

Comment:
sollen wir vllt so ein
`if (!availableImages) {
  availableImages = [...]
}`
einfügen für die default Test images?

Reply:
An der Stelle testen wir ja nur unsere Komponente, oder? Da würde ich sagen ist das nicht nötig. Oder meinst du in der Component?

Reply:
ahh hatte nicht auf dem Schirm, dass serve am Ende nicht mitgeliefert wird. Passt